modules/node/init.zsh

```diff
@@ -14,6 +14,15 @@ if [[ -s "$HOME/.nvm/nvm.sh" ]]; then
 elif (( $+commands[brew] )) && [[ -d "$(brew --prefix nvm 2>/dev/null)" ]]; then
   source $(brew --prefix nvm)/nvm.sh
 
+# Load manually installed nodenv into the shell session.
+elif [[ -s "$HOME/.nodenv/bin/nodenv" ]]; then
+  path=("$HOME/.nodenv/bin" $path)
```

Comment:
Yeah I think something more like #1001 is preferable, but with the README additions here.